lib/ui/blocs/posts_list/posts_list_state.dart

```diff
@@ -67,11 +67,16 @@ class PostsLoaded extends PostsListState {
     );
   }
 
-  /// gets all errored post
+  /// gets all errored posts
   List<Post> get getErroredPosts => posts
       .where((post) => post.status.value == PostStatusValue.ERRORED)
       .toList();
 
+  /// gets non errored posts
+  List<Post> get getPosts => posts
```

Comment:
Since this is already a getter, I think it's better to name it simply `posts` instead of `getPosts`. Maybe something more significant like `nonErroredPosts` might be better also